tests/discover/github_git_suffix/main.fmf

```diff
@@ -0,0 +1,25 @@
+test: placeholder
+framework: beakerlib
+discover:
+    how: fmf
+/detected:
+ require:
+   - library(database/mariadb)
+   - url: https://github.com/beakerlib/database.git
+
+/explicit:
+ require:
+   - library(database/mariadb)
+   - url: https://github.com/beakerlib/database.git
+     name: /mysql
+ recommend:
+   - url: https://github.com/beakerlib/database
+     name: /postgresql
+   - library(database/postgresql)
+   - url: https://github.com/psss/fmf.git
+
+
+/tmt-does-internally:
+ recommend:
+   - url: https://github.com/beakerlib/database
+     name: /postgresql
```

Comment:
Instead of directly adding tests with fake required/recommended packages, which results in extra packages to be installed on the localhost when running using `provision --how local`, it would be good to extend the existing test `/tests/discover/libraries` or run these tests in a container.